recipe/meta.yaml

```diff
@@ -8,118 +8,71 @@ package:
 source:
```

Comment:
If we do not provide a multi-output package then the package name must be `pytensor`.
But another question: do we really want to make one output instead of two?

Reply:
Do we have constraints to not build the latest version https://github.com/pymc-devs/pytensor/releases/tag/rel-2.13.1?

Reply:
It must have become available in the mean time. I will update.
I also thought about changing the name of the package to `pytensor`. @rafaelmartins was bringing up the argument that the community may be less confused if the package name stayed the same as the conda-forge one. But having refactored the recipe to this extent, I also agree it can be renamed.
As far as making one output: yes, we do. The whole point of the package having two outputs was the conda-forge maintainer fixing some runtime dependencies the wrong way, hence they created `pytensor-base` and `pytensor`. But `pytensor-base` in itself is entirely useless. This should be the right way to do it.

Reply:
got it. Anyway, this is now resolved.